### PR TITLE
Create glossary.md

### PR DESCRIPTION
**Pull request by decanus**

### CODE REVIEW

**Comment by decanus** [comment]
@oskarth Right now I just moved all definitions into the glossary, would be good to have the data-sync specific ones only in that file. Maybe you could quickly evaluate which belong where.

File: data_sync/p2p-data-sync-comparison.md [hunk 1]
```diff
@@ -24,28 +24,7 @@ In this paper we go through various approaches to data sync in a p2p context. We
...
-| Content addressable storage (CAS) | Storing information such that it can be retrieved by its content, not its location. Commonly performed by the use of cryptographic hash functions. |
-| Public P2P | Open network where peers can connect to each other. |
-| Structured P2P network | A public p2p network where data placement is related to the network topology. E.g. a DHT. |
-| Unstructed P2P network | A public p2p network where peers connect in an ad hoc manner, and a peer only knows its neighbors, not what they have. |
-| Super-peer P2P network | A non-pure p2p network, hybrid client/server architecture, where certain peers do more work. | 
-| Private P2P | Private network where peers must mutually trust each other before connecting, can either be F2F or group-based. |
-| Group-based P2P network | A private p2p network where you need some form of access to join, but within group you can connect to new peers, e.g. friend of a friend or a private BT tracker. |
+For broad defenitions please refer to the [glossary](/glossary.md), data sync specific definitions will be listed below.
```
Comment (oskarth) [inline data_sync/p2p-data-sync-comparison.md]: definitions

Also can we keep them in this doc as well for now? Can remove redundancy later, would rather it be self-contained

**Review by oskarth — COMMENTED** [review]
See in-line

**Comment by decanus** [comment]
@oskarth are there any terms you see unfit for the broader glossary document

**Comment by oskarth** [comment]
Not right now, can tweak later

**Pull request merged**